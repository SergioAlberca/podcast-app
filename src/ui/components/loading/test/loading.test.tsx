import { render, screen } from "@/test/test_utils";
import Loading from "../loading";

describe("Loading component tests", () => {
  it("render component", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });

  it("loading is in the document", () => {
    render(<Loading />);
    expect(screen.getByTestId("loading")).toBeInTheDocument();
  });
});
