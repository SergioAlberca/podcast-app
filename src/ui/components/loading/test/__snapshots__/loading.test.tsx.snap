// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loading component tests > render component 1`] = `
<div>
  <div
    class="_lds_57ddc4"
    data-testid="loading"
  >
    <div
      class="_lds__roller_57ddc4"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
